packages/cli-kit/src/version.ts

```diff
@@ -9,9 +10,10 @@ import latestVersion from 'latest-version'
  * @param name - The name of the NPM package.
  * @returns A promise to get the latest available version of a package.
  */
-export async function latestNpmPackageVersion(name: string) {
+export async function latestNpmPackageVersion(name: string, options = {force: false}) {
   debug(content`Getting the latest version of NPM package: ${token.raw(name)}`)
-  return latestVersion(name)
+  if (options.force) return latestVersion(name)
+  return cache(`latest-npm-package-version-${name}`, async () => latestVersion(name))
```

Comment:
I think it we're doing this it needs to be clearer that `cache` is a system-wide thing, not local to a project or in-memory on a process.